service/matching/physical_task_queue_manager.go

```diff
@@ -295,6 +400,10 @@ func (c *physicalTaskQueueManagerImpl) PollTask(
 
 		task.namespace = c.partitionMgr.ns.Name()
 		task.backlogCountHint = c.backlogMgr.BacklogCountHint
+		if pollMetadata.forwardedFrom == "" {
```

Comment:
I'm confused - If the task is coming from a forwarded poll, won't it be caught by the current if-condition? (since  pollMetadata.forwardedFrom == "" will be false, and we won't count that in our tracker)

Reply:
Discussed sync and see the reasoning for this.